### PR TITLE
10.0 migrate l10n es aeat sii restapi

### PR DESCRIPTION
**Pull request by ioans73**
Module l10n_es_aeat_sii_restapi migrated to V10

### CODE REVIEW

**Comment by ioans73** [comment]
Module l10n_es_aeat_sii_restapi migrated to V10

File: l10n_es_aeat_sii_restapi/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "license": "AGPL-3",
+    "depends": [
+        "l10n_es_aeat_sii",
+    ],
+    "data": [
+        'views/account_invoice_import_view.xml',
+        'views/account_invoice_import_actions.xml',
+        'security/ir.model.access.csv',
+    ],
+    "application": False,
```
Comment (fuentes73) [inline l10n_es_aeat_sii_restapi/__manifest__.py]: Quitar claves a False

File: l10n_es_aeat_sii_restapi/views/account_invoice_import_actions.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	<data>
```
Comment (fuentes73) [inline l10n_es_aeat_sii_restapi/views/account_invoice_import_actions.xml]: Quitar data de todos los XML

File: l10n_es_aeat_sii_restapi/views/account_invoice_import_view.xml [hunk 3]
```diff
@@ -0,0 +1,402 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	<data>
```
Comment (fuentes73) [inline l10n_es_aeat_sii_restapi/views/account_invoice_import_view.xml]: Quitar data

**Review by fuentes73 — CHANGES_REQUESTED** [review]

**Review by fuentes73 — APPROVED** [review]

**Pull request merged**